PointOfInteraction/SmartSpot/doc/schema.json

```diff
@@ -0,0 +1,78 @@
+{
+	"$schema": "http://json-schema.org/schema#",
+	"id": "https://fiware.github.io/dataModels/PointOfInteraction/SmartSpot/doc/schema.json",
+	"title": "FIWARE - Smart Point of Interest",
+	"description": "Smart Point of Interest",
+	"type": "object",
+	"allOf": [{
+		"$ref": "https://fiware.github.io/dataModels/common-schema.json#/definitions/GSMA-Commons"
+	}, {
+		"$ref": "https://fiware.github.io/dataModels/common-schema.json#/definitions/Location-Commons"
```

Comment:
probably you don't need location commons as the data model is not referencing any GeoJSON structure